resources/references/adr/2020-08-21-unified-notification-titles.md

```diff
@@ -81,3 +81,4 @@ It would cross the design idea of unified titles though and should only be consi
 > *When creating notifications, just decide on the correct type of notification, 
  add a meaningful message, don't waste even a thought on creating a title...
  And you're done!*
+ 
```

Comment:
@jrson83 there are still 3 changes from the adrs left. Please remove them

Reply:
> @jrson83 there are still 3 changes from the adrs left. Please remove them 

I am currently unable to operate due to physical exhaustion. I will take care of it as soon as I am able to.

Reply:
@jrson83 get well soon :sleeping_bed: :slightly_frowning_face:

Reply:
@Isengo1989 Thanks, I feel better now.

So we might have a conflict with this last file:
[resources/references/adr/2023-02-27-native-extension-system-with-vue.md](https://github.com/shopware/docs/pull/1596/files#diff-6f795a2f789718521165ae0ee07f115050c61da25e74bff16c5bb3bc464095c3)

In the original file, there is a final new line, which may also include either a space or a tab character. I'm not sure which one if it is, I will try to insert a `space` in the final new line and commit the file.

![2024-12-11_08h09_53](https://github.com/user-attachments/assets/267352b6-5326-42d5-96e3-be0e41166206)

Reply:
OK that did not work at all. It seams there is a conflict with all the 3 files, cause of different indentation, I cannot revert because the indentation is now forced by the `.editorconfig` :worried: